experiment/run_experiment.py

```diff
@@ -66,14 +66,17 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
     and returns it."""
     config = yaml_utils.read(config_filename)
     filestore_params = {'experiment_filestore', 'report_filestore'}
-    cloud_config = {'cloud_compute_zone'}
-    string_params = cloud_config.union(filestore_params)
+    cloud_config = {'cloud_compute_zone', 'cloud_project'}
+    local_config = {'docker_registry'}
+    string_params = cloud_config.union(filestore_params).union(local_config)
     int_params = {'trials', 'max_total_time'}
     required_params = int_params.union(filestore_params)
 
     local_experiment = config.get('local_experiment', False)
     if not local_experiment:
```

Comment:
nit: don't need the not, can switch the order, if local_experiment: do that else: